index.js

```diff
@@ -5,7 +5,6 @@ module.exports = function () {
     log.fatal('uncaughtException');
```

Comment:
Can you make the main exported function take an options param, like: 
```
module.exports = function (options) {
```
then do 
```
if (options.ringSystemBellOnAbort) console.log('\u0007');
```
?
This will allow us to do the following in our ./bin/bla files: 
```
require('error-trap')({ ringErrorBellOnAbort: config.NODE_ENV === 'development' });
```
That way, we don't get any weird system bell log statements or possible funky behavior in other envs.